benchmark/benchmarks.jl

```diff
@@ -0,0 +1,53 @@
+#=
+This file defines a benchmark suite with the tools provided by
+`PkgBenchmark` and `BenchmarkTools`.
+
+To run the benchmarks, execute:
+
+```julia
+julia> using PkgBenchmark
+julia> results = benchmarkpkg("LazySets")
+```
+
+To compare current version to another tagged version:
+
+```julia
+julia> results = judge("LazySets", "v1.2.0")
```

Comment:
This crashes because in that version we did not have a `benchmarks.jl` file.

Reply:
true, thanks, i changed it to `results = judge("LazySets", <tagged-version-or-branch>)`